modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java

```diff
@@ -195,42 +181,37 @@ private TestAppender createAppenderForLogMonitoring() {
 
     private void publicationServiceThrowsException() throws ApiGatewayException {
         serviceSucceedsOnGetRequest(publication);
-        when(publicationService.updatePublication(any(SortableIdentifier.class), any(Publication.class)))
+        when(publicationService.updatePublication(any(Publication.class)))
             .then((Answer<Publication>) invocation -> {
                 throw new RuntimeException(ModifyPublicationHandlerTest.SOME_MESSAGE);
             });
     }
 
     private void serviceFailsOnModifyRequestWithRuntimeError() throws ApiGatewayException {
         serviceSucceedsOnGetRequest(publication);
-        when(publicationService.updatePublication(any(SortableIdentifier.class), any(Publication.class)))
+        when(publicationService.updatePublication(any(Publication.class)))
             .thenThrow(RuntimeException.class);
     }
 
     private String serviceFailsOnGetRequestWithNotFoundError(SortableIdentifier identifier) throws ApiGatewayException {
         String expectedDetail = String.format(RESOURCE_NOT_FOUND_ERROR_TEMPLATE, identifier.toString());
-        when(publicationService.getPublication(any(SortableIdentifier.class))).thenThrow(
-            new NotFoundException(expectedDetail));
+        when(publicationService.getPublication(any(UserInstance.class), any(SortableIdentifier.class)))
+            .thenThrow(new NotFoundException(expectedDetail));
         return expectedDetail;
     }
 
     private void serviceSucceedsOnGetRequest(Publication publication) throws ApiGatewayException {
-        when(publicationService.getPublication(any(SortableIdentifier.class))).thenReturn(publication);
+        when(publicationService.getPublication(any(UserInstance.class), any(SortableIdentifier.class)))
+            .thenReturn(publication);
     }
 
     private void serviceSucceedsAndReturnsModifiedPublication(Publication modifiedPublication)
         throws ApiGatewayException {
         serviceSucceedsOnGetRequest(publication);
-        when(publicationService.updatePublication(any(SortableIdentifier.class), any(Publication.class)))
+        when(publicationService.updatePublication(any(Publication.class)))
             .thenReturn(modifiedPublication);
     }
 
-    private void serviceSucceedsOnGetRequestAndFailsOnUpdate() throws ApiGatewayException {
```

Comment:
The ErrorResponeException is not thrown by any method of the current DynamoDB service